salt/backups/unmount_drive.sls

```diff
@@ -0,0 +1,14 @@
+unmount_backup_drive:
+  mount.unmounted:
+    - name: /backups
+    - device: /dev/xvdb
+
+detach_backup_volume:
+  cloud.action:
+    - func: boto_ec2.detach_volume
+    - kwargs:
+        tags:
+          Name: {{ backup_volume_name }}
```

Comment:
This needs to be set somewhere. Right now it will throw an error for an undefined variable.